### PR TITLE
Trad_client centos7 tests

### PR DESCRIPTION
**Pull request by MalloZup**

### CODE REVIEW

File: features/centos_tradclient.feature [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2017 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: register a traditional client centos7
+
+  Scenario: Create bootstrap-repo for sle12sp2
```
Comment (mbologna) [inline features/centos_tradclient.feature]: "sle12sp2": typo?

File: features/centos_tradclient.feature [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+     Given I am authorized
+     And  I run "sed s/enabled=0/enabled=1/g /etc/yum.repos.d/Devel_Galaxy_Manager_Head_RES-Manager-Tools-7-x86_64.repo  -i" on "ceos-minion" without error control
+     And  I run "yum install -y rhn-client-tools rhn-check rhn-setup rhnsd hwdata  m2crypto wget osad" on "ceos-minion"
```
Comment (mbologna) [inline features/centos_tradclient.feature]: double space between `hwdata` and `m2crypto`

File: features/centos_tradclient.feature (hunk 1 above)
Comment (MalloZup) [inline features/centos_tradclient.feature]: copy and paste

File: features/centos_tradclient.feature (hunk 2 above)
Comment (MalloZup) [inline features/centos_tradclient.feature]: ok thx

**Comment by MalloZup** [comment]
http://www.spacewalkproject.org/documentation/api/2.3/handlers/SystemHandler.html#schedulePackageInstall

**Pull request merged**